src/d3a/models/strategy/predefined_pv.py

```diff
@@ -172,3 +172,45 @@ def area_reconfigure_event(self, **kwargs):
         if key_in_dict_and_not_none(kwargs, 'power_profile'):
             self._power_profile_W = kwargs['power_profile']
         self.read_config_event()
+
+
+class PVForecastStrategy(PVPredefinedStrategy):
```

Comment:
Same as above, we'll never need to instantiate it without an external connection